### ISSUE TITLE
Epic games version ;)

### ISSUE THREAD

**Issue by Hymen065** [xNVSE/NVSE#166]
NVSE does not support epic version of FNV yet, is there anyway to have the epic version of NVSE?

**Comment by WallSoGB** [comment]
No, EXE is recompiled, and no one is going to decode the game again and update all mods. Thank Bethesda.

**Comment by Sysadminfromhell** [comment]
So this mod is not going to be Updated?

**Comment by WallSoGB** [comment]
Can't be due to reasons mentioned above.

**Comment by bluefieldcreator** [comment]
Truly unfortunate to the many new players that got the game through Epic Games, there's an ongoing sale as of yesterday that made the game free.

**Comment by Buffloli** [comment]
So....just don't play the Epic Games version?

**Comment by XDaron** [comment]
So, I found a solution to this, what you need to do is download the steam_api.dll and the FalloutNV.exe from steam and paste it on your Fallout New vegas English Folder, then run the NVSE and it should work, I had the original New vegas without expansion on steam so I just tranfered these files to my epic games version and it worked for me, running the game with mods and the DLC message also appeared so I assume everything is fine

there is probably a safer way to do this, and I reccomend you dont, but I will share both the files here , again, you shouldn't trust a random EXE someone post online, but if you're feeling trusting... https://drive.google.com/drive/folders/1UBms_mX6Me4Q_Wv-hty7xozP7citJOo9?usp=sharing

**Comment by VishVadeva** [comment]
Are you stupid?

**Comment by XDaron** [comment]
well, shit, just know that if you have FalloutNV.exe and Steamapi.dll inside your fallout:new vegas english folder, it should work, sorry for the link

**Comment by WallSoGB** [comment]
tldr if you own a Steam version of the game, it works????/

**Comment by onv-il** [comment]
Please include a "soyjak" in one of your future cracks. Preferably a "Cobson" or "Feraljak" since they are some of my favorites! :)

**Comment by AVeryUncreativeUsername** [comment]
Hey guys. Don't forget about me! I want to shitpost too! And Demo. I know you're reading this. I still need that new xNVSE update. No rush. Just letting you know.

**Comment by Hymen065** [comment]
Oh....... So thats mean all the NV mods in nexus are not gona work on the epic version of NV? Instead, they only work on gog or steam version, right? ;(

**Comment by RedSnt** [comment]
Sad to hear about the EGS version. Not for me, I own both the GOG and Steam version, but for everyone else picking up the game right now. But fully understandable that you'll not be going through the EGS executable, from scratch, all over again with the progress already done on the other versions.

**Comment by Sysadminfromhell** [comment]
Is there any way to help maybe or decombile the exe orself to provied the needed help? I would love to have some mods for the game because its the ultimate edition with all dlcs :(

**Comment by carlop3333** [comment]
> Is there any way to help maybe or decombile the exe ourself to provied the needed help? I would love to have some mods for the game because its the ultimate edition with all dlcs :(

There is a new patcher going around that downgrades the epic's exe (no achievements), already tried it and works flawlessly

**Comment by Space734** [comment]
> > Is there any way to help maybe or decombile the exe ourself to provied the needed help? I would love to have some mods for the game because its the ultimate edition with all dlcs :(
> 
> There is a new patcher going around that downgrades the epic's exe (no achievements), already tried it and works flawlessly

im curious  where i can find it

**Comment by AVeryUncreativeUsername** [comment]
I don't think Stewie wants you redistributing his patcher. It was removed for a reason. When I crossed the Stewster I paid dearly for it.

**Comment by carlop3333** [comment]
> I don't think Stewie wants you redistributing his patcher. It was removed for a reason. When I crossed the Stewster I paid dearly for it.

Deleted the comment, thank you for warning that, but anyways, that is the same patcher in [reddit](https://old.reddit.com/r/Fallout/comments/13urhzp/how_to_mod_fallout_new_vegas_on_the_epic_games/) and it appears in every single yt video about that lol

**Comment by AVeryUncreativeUsername** [comment]
He is currently working on a new one. So do keep an eye out.

**Comment by Demorome** [comment]
Since Stewie released a new patcher a bit ago, closing this issue.
